pkg/apis/serving/v1beta1/register.go

```diff
@@ -24,9 +24,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// GroupNamePrefix is the prefix for label key and annotation key
-const groupNamePrefix = serving.GroupName + "/"
```

Comment:
@vagababov : I am deleting the reference to `groupNamePrefix` variable . Previously it made sense to declare as private var as it was not used outside the package but  thats not true anymore